src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
-   in this sample. The number of elements in this array should be equal to the ploidy at
-   this site, plus 1.
+   Phred-scaled genotype likelihoods rounded to the closest integer, and otherwise defined precisely
+   as the genotype likelihoods field. VCF genotype field reserved key PL, Number=G.
    */
-  array<float> genotypePosteriors = [];
+  array<int> phredScaledLikelihoods = [];
 
+  // PQ Number=1 Type=Integer
   /**
-   The log-odds ratio of being a true vs. false variant under a trained statistical model.
-   This model can be a multivariate Gaussian mixture, support vector machine, etc.
+   Phred-scaled probability that alleles are ordered incorrectly, against all other members in
+   the same phase set. VCF genotype field reserved key PQ.
    */
-  union { null, float } vqslod = null;
+  union { null, int } phaseSetQuality = null;
 
-  /**
-   If known, the feature that contributed the most to this variant being classified as
-   a false variant.
-   */
-  union { null, string } culprit = null;
 
+  // SB is not a reserved key in VCF v4.2 or v4.3
   /**
-   Additional feature info that doesn't fit into the standard fields above.
-   They are all encoded as (string, string) key-value pairs.
+   Per-sample component statistics which comprise the Fisher's Exact Test to detect strand bias.
+   If set, the array should contain the number of reads that support the reference allele on the
+   forward strand, the reference allele on the reverse strand, the alternate allele on the forward
+   strand, and the alternate allele on the reverse strand, in that order. VCF genotype field key SB.
    */
-  map<string> attributes = {};
-}
+  //array<int> strandBias = [];  Genotype.strandBiasComponents previously
 
-/**
- Genotype.
- */
-record Genotype {
 
-  /**
-   The variant called at this site.
-   */
-  union { null, Variant } variant = null;
+  //
+  // Not present in VCF specification yet present in Genotype
 
-  /**
-   The reference contig that this genotype's variant exists on.
-   */
-  union { null, string } contigName = null;
+  // The minimum number of reads seen at this site across samples when joint calling variants.
+  // Analogous to VCF's MIN_DP.
+  //union { null, int } minReadDepth = null;
 
-  /**
-   The 0-based start position of this genotype's variant on the reference contig.
-   */
-  union { null, long } start = null;
+  // Log scaled likelihoods that we have n non-reference alleles at this site. The number of
+  // elements in this array should be equal to the ploidy at this site, plus 1.
+  //array<float> nonReferenceLikelihoods = [];
 
-  /**
-   The 0-based, exclusive end position of this genotype's variant on the reference contig.
-   */
-  union { null, long } end = null;
+  // We split multi-allelic VCF lines into multiple single-alternate records. This bit is set
+  // if that happened for this record.
+  //union { boolean, null } splitFromMultiAllelic = false;
```

Comment:
Ah sorry, you're right. What's the rationale for removing this here? If you don't have this field and you have a triallelic site that is `GT=1/1` in a sample, you don't know that there's a second alt at the site from looking at the genotype record.